server/services/bigtable.py

```diff
@@ -48,19 +58,36 @@ def get_project_id():
   return project_id
 
 
-async def write_row(session_info):
+async def write_feedback(session_info, data):
```

Comment:
Does this need async?

Reply:
Removed async. And also return 500 if this failed.